src/Compilers/CSharp/Portable/Lowering/LocalRewriter/LocalRewriter.cs

```diff
@@ -25,9 +25,9 @@ internal sealed partial class LocalRewriter : BoundTreeRewriterWithStackGuard
         private readonly SyntheticBoundNodeFactory _factory;
         private readonly SynthesizedSubmissionFields _previousSubmissionFields;
         private readonly bool _allowOmissionOfConditionalCalls;
-        private readonly LoweredDynamicOperationFactory _dynamicFactory;
+        private LoweredDynamicOperationFactory _dynamicFactory;
         private bool _sawLambdas;
-        private bool _sawLocalFunctions;
+        private int _availablelocalFunctionOrdinal;
```

Comment:
nit: it seems like this variable should be named `_availableLocalFunctionOrdinal`

Reply:
> it seems like this variable should be named _availableLocalFunctionOrdinal

I'll take care of this in a follow up PR.

---
In reply to: [538671479](https://github.com/dotnet/roslyn/pull/49802#discussion_r538671479) [](ancestors = 538671479)